README.rst

```diff
@@ -4,7 +4,7 @@
 
 Swiss-army knife to synchronize Postgres roles and ACLs from any LDAP directory.
 
-.. _documentation: http://ldap2pg.readthedocs.io/
+.. _documentation: https://ldap2pg.readthedocs.io/en/latest/
```

Comment:
👍 for HTTPS

However, i prefer not to enforce language in README and let RTD content negociation do the redirection. What do you think of this ?

Reply:
We'll see that if ever we translate the doc.

Reply:
Oh.. Didn't think about that language issue.. Yeah, it's a good idea to take care of it when we have different translations available..

Thanks for accepting the pull request :smile: